### DISCUSSION TITLE
Imagen choice not preview

### DISCUSSION THREAD

**Discussion by jlop77** [Lumieducation/Lumi#2068] [Lumi Run]
Create a "image choice" but it does not show anything in Preview. I also tried downloading a .hp5 from H5p.org to see if it worked and does not work either the Preview.

I tried to export to a .html file, hoping to see if I was exported the content, but when opening it in the browser does not show anything.

I do this from an installation in wordpress and the preview works.

Aclaro, esto solo me sucedido hasta ahora, con la "imagen choice".

What can be happening, with Lumi preview?

**Comment by sr258** [comment]
Hi, this is a know bug that is caused by an oversight by the creators of the content type. It uses the human-readable name of the library and compares it with the hardcoded string "Image". I assume that you have set Lumi to a language other than English, so the human readable-name of the library is not "Image" and the content type breaks. @otacke has submitted a PR for this a long time ago, but unfortunately Joubel hasn't found the time to merge it yet.

An easy workaround is to temporarily change the language of Lumi to English, preview or export the content type and then change it back to your regular language.

**Reply by jlop77** [reply]
I appreciate your prompt response.

Effectively, I placed English and it works for me.

Thank you.
